portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -38,18 +38,18 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
     boolean loginStatus = userService.isUserLoggedIn();
+    String urlToRedirectToAfterUserLogsInOrOut = "/contact.html";
```

Comment:
nit: this is a very long variable name. "urlToRedirect" should be sufficient.

Reply:
I agree, it's very cumbersome.